angular.json

```diff
@@ -46,7 +45,13 @@
               "includePaths": ["./projects/assets-library/assets/styles", "./node_modules"]
             },
             "scripts": [],
-            "allowedCommonJsDependencies": ["graphql-tag", "zen-observable"]
+            "allowedCommonJsDependencies": ["graphql-tag", "zen-observable"],
```

Comment:
```
Option "extractCss" is deprecated: Deprecated since version 11.0. No longer required to disable CSS extraction for HMR.
```
I see this warning.

Reply:
where?

Reply:
console. when you do ng build or serve